src/components/layout/header.tsx

```
@@ -24,65 +184,246 @@ export const Header = () => {
+                onClick={() => navigate(ROUTES.MY_LISTINGS)}
+                className="rounded-lg bg-blue-600 px-4 py-2 text-sm font-medium text-white transition hover:bg-blue-700 sm:px-8 sm:py-2.5"
               >
-                Demo Login
+                Add listing
+              </button>
+              <button
+                onClick={() =>
+                  isAuthenticated ? navigate(ROUTES.PROFILE) : handleDemoLogin()
+                }
+                className="rounded-lg p-1 transition hover:bg-gray-100"
+              >
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)